### PR TITLE
Adjusted UI layout and navbar to look as expected

### LINKED ISSUE #50: Fix mobile UI

### PR DESCRIPTION
**Pull request by Nephim**

### CODE REVIEW

**Review by Gormiee — APPROVED** [review]
Very Nice. LGTM

File: src/UICrafter.Mobile/Components/Layout/MainLayout.razor [hunk 1]
```diff
@@ -11,35 +11,13 @@
...
-    <AuthorizeView>
-        <Authorized>
-            <footer class="footer">
-                <NativeNavbar />
-            </footer>
-        </Authorized>
-    </AuthorizeView>
+    <footer>
+        <NativeNavbar Class="native-navbar-fixed" />
+        <NativeNavbar Style="visibility: hidden;" />
```
Comment (Aagaard00) [inline src/UICrafter.Mobile/Components/Layout/MainLayout.razor:20]: Why 2 of them?

**Review by Aagaard00 — APPROVED** [review]
LGTM

File: src/UICrafter.Mobile/Components/Layout/MainLayout.razor (hunk 1 above)
Comment (Nephim) [reply src/UICrafter.Mobile/Components/Layout/MainLayout.razor:20]: This is to offset the body with the size of the NativeNavBar Alternative to this is that I would need to add margin to the body or something but I dont know about the exact size so I think a hidden NavBar is a better choice.

**Pull request merged**